### PR TITLE
Feature/347 opengraph tags

### PR DESCRIPTION
**Pull request by Niekvandam**
## Description
Updated open graph tags for better website preview

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [X] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I updated the changelog with an end-user readable description
-   [ ] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

1. Serve the website with ng serve
2. Tunnel the website to a public URL (localtunnel, ngrok or any other tunneling service of choice )
3. Send the public website URL in a DM (discord / slack / Whatsapp etc.) and check for the preview of DeX.  

## Link to issue

Closes: #347

### CODE REVIEW

**Review by Brend-Smits — DISMISSED** [review]

File: src/index.html [hunk 1]
```diff
@@ -35,13 +35,15 @@
   <meta name="msapplication-TileColor" content="#da532c">
   <meta name="msapplication-config" content="assets/favicon/browserconfig.xml">
   <meta name="theme-color" content="#ffffff">
+  <meta property="og:image" content="https://dex.software/assets/images/dex-logo-dark-orange.png"/>
```
Comment (Ruby77) [inline src/index.html:38]: https://ogp.me/ I read the following:
`og:image:url - Identical to og:image.`
`(...)`
`og:image:alt - A description of what is in the image (not a caption). If the page specifies an og:image it should specify og:image:alt.`

Shouldn't we have the last tag as well?

**Review by Ruby77 — CHANGES_REQUESTED** [review]
One small question/note... 😄

File: src/index.html (hunk 1 above)
Comment (Niekvandam) [reply src/index.html:38]: Definitely, I totally looked over this comment!

Updating it rn

**Review by Ruby77 — APPROVED** [review]
Awesome!

**Review by Brend-Smits — APPROVED** [review]

**Pull request merged**